py/src/braintrust/imported_types.py

```diff
@@ -0,0 +1,185 @@
+"""Re-exports generated typespecs that are considered public API."""
```

Comment:
I'd be ok if this was `types.py` and we had a `_generated_types.py`

Reply:
Yeah I wanted to avoid using the name `types.py` because it felt too general. I'm fine with renaming everything to `generated_types`.